src/micromamba/install.sh

```diff
@@ -1,41 +1,50 @@
 #!/usr/bin/env bash
 
 VERSION=${VERSION:-"latest"}
+# Add destination parameter here?
 
-USERNAME=${USERNAME:-"automatic"}
+# # This looks like it's not needed:
+# USERNAME=${USERNAME:-"automatic"}
 
 set -e
 
+# Is this the standard for dev-container features? It seems like this way each
+# feature will rerun `apt-get update` which seems inefficient. But I suppose
+# this is to prioritize reliability?
```

Comment:
The installation may fail if we do not run `apt-get update` because the cache status of the base image is not guaranteed.
See devcontainers/features#210